src/Update.cpp

```diff
@@ -1353,8 +1353,8 @@ static void SusceptibleToRecovered(int cellIndex)
 
 	// assert values are non-negative
 	assert(Cells[cellIndex].S >= 0);
-	assert(Cells[cellIndex].latent >= 0);
-	assert(Cells[cellIndex].infected >= 0);
+	assert(Cells[cellIndex].latent >= static_cast<int*>(0));
+	assert(Cells[cellIndex].infected >= static_cast<int*>(0));
```

Comment:
Thanks, but this doesn't completely fix the issue. 
If the pointer is decremented too far, the memory address is still positive. 
It would be safer to remove these asserts against the int* pointers - they can be re-added in a future PR if necessary.

Reply:
Makes sense. Removed as requested. Thanks for the review.

Reply:
Removing is fine for this PR

I think ultimately the right check is to test that `Cells[cellIndex].infected` points somewhere within the range [`Cells[cellindex].susceptible`, Cells[cellindex].susceptible + Cells[cellindex].n`).